### ISSUE TITLE
Every page should include a single H1 title

### ISSUE THREAD

**Issue by jeromehardaway** [Vets-Who-Code/vets-who-code-app#358]
Requirement: An H1 title provides information to blind-users using screen-readers of what the main topic of the page is and each page should have exactly one H1 title.

On the board page find the extra H1's and change them.
